metaflow/cli.py

```diff
@@ -101,6 +110,82 @@ def logger(body='', system_msg=False, head='', bad=False, timestamp=True):
                 fg=LOGGER_BAD_COLOR if bad else None)
 
 
+def get_ordered_steps(graph, type='linear'):
```

Comment:
please document a TODO to support the other Metaflow graphs.

Reply:
done